app/services/registration_service.rb

```diff
@@ -1,7 +1,10 @@
 class RegistrationService
   def from_invitation(email, invitation_token, user_data = {})
-    Invitation.find_by(email: email, token: invitation_token)
-    User.create({ email: email }.merge(user_data))
+    invitation = Invitation.find_by(email: email, token: invitation_token)
```

Comment:
why not maintaining 'em, but it should be marked as used somehow.
setting an `activated_at` field to the current time, could be nice. what do you think?